images/linux/scripts/installers/aws-iam-authenticator.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash -e
+################################################################################
+##  File:  aws-iam-authenticator.sh
+##  Desc:  Installs AWS IAM Authenticator
+################################################################################
+
+source $HELPER_SCRIPTS/install.sh
+
+# Determine latest aws-iam-authenticator CLI version
+AWS_IAM_AUTHENTICATOR_LATEST_VERSION_URL=https://api.github.com/repos/kubernetes-sigs/aws-iam-authenticator/releases/latest
+AWS_IAM_AUTHENTICATOR_DOWNLOAD_URL=$(curl -s $AWS_IAM_AUTHENTICATOR_LATEST_VERSION_URL | jq -r '.assets[].browser_download_url | select(contains("linux_amd64"))')
+AWS_IAM_AUTHENTICATOR_ARCHIVE=$(basename $AWS_IAM_AUTHENTICATOR_DOWNLOAD_URL)
+
+# Install aws-iam-authenticator CLI
+cd /tmp
+download_with_retries $AWS_IAM_AUTHENTICATOR_DOWNLOAD_URL
+mv $AWS_IAM_AUTHENTICATOR_ARCHIVE /usr/local/bin 
```

Comment:
Maybe
```
download_with_retries $AWS_IAM_AUTHENTICATOR_DOWNLOAD_URL /tmp $AWS_IAM_AUTHENTICATOR_ARCHIVE
chmod +x /tmp/$AWS_IAM_AUTHENTICATOR_ARCHIVE && mv /tmp/$AWS_IAM_AUTHENTICATOR_ARCHIVE /usr/local/bin/aws-iam-authenticator
```
since we know the name and chmod +x should be invoked to be able to run the binary.

Reply:
sounds good to me.